src/index.html

```diff
@@ -1,22 +1,446 @@
+            <h2 class="section-title">
+              Sound. Vision.
+              <br />
+              It’s your pick.
+            </h2>
+            <div class="categories__wrapper">
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/portable-speakers-a.png"
+                      alt="Portable Speakers photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/portable-speakers-b.png"
+                      alt="Portable Speakers photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Portable speakers</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-a.png"
+                      alt="Earphones photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-b.png"
+                      alt="Earphones photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Earphones</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-A.png"
+                      alt="Smart Home photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-b.png"
+                      alt="Smart Home photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+
+              <div class="category__button">
+                <button class="button">
```

Comment:
this button is not styled well

verify outline property of the button

the ui of button "send" is correct